sdcm/cluster.py

```diff
@@ -3610,6 +3610,9 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600,
         if wait_db_up:
             node.wait_db_up(verbose=verbose, timeout=timeout)
             nodes_status = node.get_nodes_status()
+            # only verify status of current setup node, other nodes might be in UJ status
+            if Setup.USE_LEGACY_CLUSTER_INIT:
+                nodes_status = {node.ip_address: nodes_status[node.ip_address]}
```

Comment:
@amos, when this could happened?

Reply:
Use default use_legacy_cluster_init,   legacy cluster_init will be used.
If there are multiple non-seeds are initing in parallel, the problem will occur.

Reply:
btw, you tagged a wrong name ;-)   @amoskong